plugins/quickinspector/quickinspector.cpp

```diff
@@ -387,6 +389,28 @@ void QuickInspector::objectSelected(void *object, const QString &typeName)
         selectSGNode(reinterpret_cast<QSGNode *>(object));
 }
 
+void QuickInspector::objectCreated(QObject *object)
+{
+    if (!object)
```

Comment:
unnecessary check, Probe guarantees object to not be null, and the next if() tests the same implicitly as well